tests/smc/test_smc.py

```diff
@@ -79,7 +80,9 @@ def test_sample(self):
         initial_rng_state = np.random.get_state()
         with self.SMC_test:
             mtrace = pm.sample_smc(
-                draws=self.samples, return_inferencedata=False, progressbar=not _IS_WINDOWS
+                draws=self.samples,
+                return_inferencedata=False,
+                progressbar=not _IS_WINDOWS,
```

Comment:
your PR is introducing several unrelated formatting changes by adding the trailing comma after the last argument. Can you revert those?